its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/Tests.java

```diff
@@ -51,7 +51,7 @@ public void afterAll(ExtensionContext extensionContext) throws Exception {
   }
 
   private static Orchestrator createOrchestrator() {
-    var version = System.getProperty("sonar.runtimeVersion", "DEV[10.8]");
+    var version = System.getProperty("sonar.runtimeVersion", "DEV");
```

Comment:
To not forget about it.

[SCAN4NET-168](https://sonarsource.atlassian.net/browse/SCAN4NET-168) is also about updating `DEV[10.8]` to `DEV` in `azure-pipeline.yml` (there is at least one occurrence).